requirements/constraints.txt

```diff
@@ -149,3 +149,8 @@ PyYAML==3.12 \
     --hash=sha256:0c507b7f74b3d2dd4d1322ec8a94794927305ab4cebbe89cc47fe5e81541e6e8 \
     --hash=sha256:b4c423ab23291d3945ac61346feeb9a0dc4184999ede5e7c43e1ffb975130ae6 \
     --hash=sha256:5ac82e411044fb129bae5cfbeb3ba626acb2af31a8d17d175004b70862a741a7
+attrs==17.3.0 \
+    --hash=sha256:e7d51b70f19a4da5fe6b3c9938983e0af3b91e230edc504bd73c443d98037063 \
+    --hash=sha256:c78f53e32d7cf36d8597c8a2c7e3c0ad210f97b9509e152e4c37fa80869f823c
```

Comment:
Yay! Now we should just start using attrs with reckless abandon.

Reply:
Well, I mean, add it to `requirements/default.txt` first. But yes.